src/org/labkey/test/pages/core/admin/AuditLogMaintenancePage.java

```diff
@@ -0,0 +1,83 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class AuditLogMaintenancePage extends LabKeyPage<AuditLogMaintenancePage.ElementCache>
+{
+    public AuditLogMaintenancePage(WebDriver driver)
+    {
+        super(driver);
+        waitForPage();
+    }
+
+    public static AuditLogMaintenancePage beginAt(WebDriverWrapper wrapper)
+    {
+        wrapper.beginAt(WebTestHelper.buildURL("professional", "auditLogMaintenance"));
+        return new AuditLogMaintenancePage(wrapper.getDriver());
+    }
+
+    @Override
+    public void waitForPage()
+    {
+        waitFor(() -> {
+                    try
+                    {
+                        return elementCache().retentionTime.isDisplayed();
+                    }
+                    catch (NoSuchElementException nse)
+                    {
+                        return false;
+                    }
+                },
+                "The audit log maintenance page did not load in time.",
+                5_000);
+    }
+
+    @Override
+    protected AuditLogMaintenancePage.ElementCache newElementCache()
+    {
+        return new AuditLogMaintenancePage.ElementCache();
+    }
+
+    public String getRetentionTime()
+    {
+        return elementCache().retentionTime.getText();
+    }
+
+    public AuditLogMaintenancePage setRetentionTime(String value)
+    {
+        selectOptionByText(elementCache().retentionTime, value);
+        return this;
+    }
+
+    public AuditLogMaintenancePage setExportDataBeforeDeleting(boolean value)
+    {
+        setCheckbox(elementCache().exportData, value);
+        return this;
+    }
+
+    public void clickSave()
+    {
+        elementCache().saveBtn.click();
+    }
+
+    public void clickCancel()
+    {
+        elementCache().cancelBtn.click();
+    }
+
+    protected class ElementCache extends LabKeyPage.ElementCache
+    {
+        WebElement retentionTime = Locator.name("retentionTime").findWhenNeeded(this);
```

Comment:
Use component for select.
```suggestion
        Select retentionTime = SelectWrapper.Select(Locator.name("retentionTime")).findWhenNeeded(this);
```